src/poliastro/tests/tests_twobody/test_orbit.py

```diff
@@ -211,3 +211,21 @@ def test_hyperbolic_nu_value_check():
     assert isinstance(positions, CartesianRepresentation)
     assert isinstance(values, Time)
     assert len(positions) == len(values) == 100
+
+
+def test_hyperbolic_modulus_wrapped_nu():
```

Comment:
This test is a bit complicated though. What about testing that the first `position` of `a.sample(100)` is exactly the position of `a`, for this particular case?

Reply:
Yeah! I will do that ! Thanks :)

Reply:
With your permission, I simplified it even further :)

Reply:
I didn't knew much about AstroPy. I will work on it too! Thanks :)